website/static/assets/js/chart_scripts.js

```diff
@@ -1,219 +1,245 @@
-
```

Comment:
Can you explain all these changes in the JS files? I know most of them are design changes. But I think you made some changes in the plots and I'm scared that may break something.

Reply:
It looks like I made changes to the plots because of copying and pasting, but the only change I made was making the notifications go away after 5 seconds